configs/common/modules.yaml

```diff
@@ -196,7 +196,7 @@ modules:
         environment:
           set:
             'NEMSIO_INC': '{prefix}/include'
-            'NEMSIO_LIB': '{prefix}/lib'
+            'NEMSIO_LIB': '{prefix}/lib64/libnemsio.a'
```

Comment:
What downstream changes does this require?

Reply:
I'm not aware of any codes currently using this variable. Most codes use CMake and so don't use this variable. The only place I'm aware of this variable being used is GLDAS (which still uses hpc-stack), so this change is basically to make it usable by GLDAS.